src/docs/references/05-availability/index.md

```diff
@@ -98,13 +98,11 @@ availability plan for a listing can be found in the
 [time-based bookings guide](/guides/how-to-take-time-based-bookings-into-use/).
 
 > **Note:** By default,
-> [FTW](https://github.com/sharetribe/flex-template-web/) is using
+> [Flex Template for Web](https://github.com/sharetribe/flex-template-web/) uses
 > [day-based availability](https://www.sharetribe.com/docs/references/availability/#day-based-availability-management).
-> If you are planning to use
-> [time-based availability](https://www.sharetribe.com/docs/references/availability/#time-based-availability-management)
-> in your marketplace, you can start with
+> If you want to use in your marketplace instead, it's recommended that you start with
```

Comment:
This has lost something relevant. "[time-based availability](https://www.sharetribe.com/docs/references/availability/#time-based-availability-management)"

Reply:
"time-based booking process"